x-pack/plugin/searchable-snapshots/qa/s3/src/javaRestTest/java/org/elasticsearch/xpack/searchablesnapshots/s3/S3SearchableSnapshotsCredentialsReloadIT.java

```diff
@@ -128,9 +122,9 @@ public void testReloadCredentialsFromAlternativeClient() throws IOException {
 
         repositoryAccessKey = accessKey1;
         keystoreSettings.put("s3.client.default.access_key", accessKey1);
-        keystoreSettings.put("s3.client.default.secret_key", randomIdentifier());
+        keystoreSettings.put("s3.client.default.secret_key", randomSecretKey());
         keystoreSettings.put("s3.client." + alternativeClient + ".access_key", accessKey2);
-        keystoreSettings.put("s3.client." + alternativeClient + ".secret_key", randomIdentifier());
+        keystoreSettings.put("s3.client." + alternativeClient + ".secret_key", randomSecretKey());
         cluster.updateStoredSecureSettings();
         assertOK(client().performRequest(new Request("POST", "/_nodes/reload_secure_settings")));
```

Comment:
The reload call needs to provide keystore password when in FIPS mode similar to [this](https://github.com/elastic/elasticsearch/blob/b563145129ce5a85428e388c3929c79ed0f59fd6/modules/repository-s3/src/javaRestTest/java/org/elasticsearch/repositories/s3/RepositoryS3RestReloadCredentialsIT.java#L110-L113).

Reply:
Ah sorry for the noise I thought this had passed the FIPS tests when I asked for your review. Fixed that now.